components/brave_federated/tasks_constants.h

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_BRAVE_FEDERATED_TASKS_CONSTANTS_H_
+#define BRAVE_COMPONENTS_BRAVE_FEDERATED_TASKS_CONSTANTS_H_
+
+namespace brave_federated {
+
+constexpr char kAdNotificationTaskName[] = "ad_notification_timing_task";
+constexpr int kAdNotificationTaskId = 0;
```

Comment:
Ads has been refactored to use `NotificationAd` for consistency with other ad types. Can you please update federated/covariates to use `NotificationAd`, thanks